client/starwhale/core/dataset/store.py

```diff
@@ -315,10 +317,14 @@ def from_data_link_uri(cls, data_link: Link, auth_name: str) -> ObjectStore:
             raise FieldTypeOrValueError("data_link is empty")
 
         # TODO: support other uri type
-        if data_link.uri.startswith("s3://"):
+        if data_link.scheme in ["s3", "minio", "oss", "aliyun"]:
```

Comment:
`if data_link.scheme in ["s3", "minio", "oss", "aliyun"]:` --> `if data_link.scheme in ("s3", "minio", "oss", "aliyun"):`

tuple is the better choice in this scenario.

Reply:
done